externs/html5.js

```diff
@@ -3242,3 +3242,104 @@ function HTMLTemplateElement() {}
  * @type {!DocumentFragment}
  */
 HTMLTemplateElement.prototype.content;
+
+
+/**
+ * @constructor
+ * @extends {Worker}
+ * @see http://www.w3.org/TR/service-workers/#service-worker-obj
+ */
+function ServiceWorker() {}
```

Comment:
You are correct. I thought that might be allowed in externs, but no:

> JSC_CONFLICTING_EXTENDED_TYPE: ServiceWorker cannot extend this type; interfaces can only extend interfaces at line 11 character 0
function ServiceWorker() {}